renovate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "extends":[
+     "config:base"
+  ],
+  "timezone":"America/Los_Angeles",
+  "schedule":[
+     "after 3am on the 1st and 3rd day instance on Monday"
+  ],
+  "enabledManagers":[
+    "npm"
+ ],
+ "vulnerabilityAlerts": {
+   "labels": ["security"],
+   "schedule": ["at any time"]
+ },
+  "rangeStrategy":"replace",
+  "npm":{
+     "stabilityDays": 2
+  },
+  "ignoreUnstable":true,
+  "labels": ["dependency-upgrade"],
```

Comment:
nit: I think this is the label Dependabot uses and repos have as an option when the repo is first created:

```suggestion
  "labels": ["dependencies"],
```